src/style.css

```
@@ -1,3 +1,67 @@
 body {
   margin: 0;
+  padding: 0;
+}
+
+@media (max-width: 1200px) {
+  .header {
+    padding: 10px;
+  }
+}
```

No need to write media here
```suggestion
.header {
  padding: 10px;
}
```